src/pages/Overview/index.js

```diff
@@ -1,16 +1,680 @@
-import React, { Component } from 'react';
-// import { routerRedux } from 'dva/router';
+import React from 'react';
```

Comment:
Seeing the following error on Overview page load: 

```
TypeError: Cannot read property 'forEach' of undefined
Cell
./src/pages/Overview/index.js:518
  515 | const row = cell.row.original;
  516 | let isSeen = false;
  517 | if (seenResults !== []) {
> 518 |   seenResults.forEach(item => {
      | ^  519 |     if (item.original.key === row.key) {
  520 |       isSeen = true;
  521 |     }
```

Reply:
This is essentially caching issue. Can not re-generate the error.